src/Microsoft.AspNetCore.ApplicationInsights.HostingStartup/ApplicationInsightsLoggerConfiguration.cs

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.AspNetCore.ApplicationInsights.HostingStartup
+{
+    internal class ApplicationInsightsLoggerConfiguration
+    {
+        private const string ApplicationInsightsLoggerFactory = "Microsoft.ApplicationInsights.AspNetCore.Logging.ApplicationInsightsLoggerProvider";
+        private const string ApplicationInsightsLoggerLevelSection = "Logging:" + ApplicationInsightsLoggerFactory + ":LogLevel";
+        private const string ApplicationInsightsSettingsFile = "ApplicationInsights.settings.json";
+
+        private static readonly KeyValuePair<string, LogLevel>[] _defaultLoggingLevels = {
+            new KeyValuePair<string, LogLevel>("Microsoft", LogLevel.Warning),
+            new KeyValuePair<string, LogLevel>("System", LogLevel.Warning),
+            new KeyValuePair<string, LogLevel>(null, LogLevel.Information)
+        };
+
+        public static void ConfigureLogging(IConfigurationBuilder configurationBuilder)
+        {
+            // Skip adding default rules when debugger is attached
+            // we want to send all events to VS
+            if (!Debugger.IsAttached)
```

Comment:
Unfortunately this is the only way to do it for preview1, we are going to review filter configuration code after making changes to logging in preview2.